src/faq/index.md

```diff
@@ -2,6 +2,7 @@
 title: FAQ
 type: faq
 layout: page
+toc: true
```

Comment:
Perhaps, but yeah later. At least for the FAQ, it makes sense, because readers will generally be looking for a question they know beforehand rather than reading the FAQ straight through. And it helps to read the TOC to get an idea of what's to come.